opentelemetry-sdk/src/opentelemetry/sdk/trace/export/__init__.py

```diff
@@ -266,7 +267,15 @@ class ConsoleSpanExporter(SpanExporter):
     spans to the console STDOUT.
     """
 
+    def __init__(
+        self,
+        out: typing.IO = sys.stdout,
+        formatter: typing.Callable[[Span], str] = str,
+    ):
+        self.out = out
+        self.formatter = formatter
+
     def export(self, spans: typing.Sequence[Span]) -> SpanExportResult:
         for span in spans:
-            print(span)
+            self.out.write(self.formatter(span))
```

Comment:
```suggestion
            self.out.write(format(span, self.format))
```

Reply:
I'm not sure format strings are the best fit for everything all use cases. As an example, one of the fields in the dictionary structure we need to output is the latency of the span in milliseconds which we calculate via `round((end - start) / scale)`. In other words, we're doing data transforms rather than just formatting how existing objects are output.

I suppose it's not impossible to do this sort of thing in a format string but it quickly becomes fairly hard to read compared to a formatter function which takes a `Span` and returns a `str`. So passing a function seems more succinct and flexible than format strings. Are there any major advantages you see with the format string approach? Or perhaps any issues with the formatter function approach?